functions.j2

```diff
@@ -0,0 +1,93 @@
+# Color code lookups
```

Comment:
jinja comments for what you are doing would be nice

Reply:
Done.